test/regress/cli/regress0/bv/overflow/ssubo1.smt2

```diff
@@ -1,5 +1,8 @@
 ; EXPECT: unsat
+; COMMAND-LINE: 
+; COMMAND-LINE: --solve-bv-as-int=sum
 (set-logic QF_BV)
+(declare-const u (_ BitVec 6))
 (declare-const v (_ BitVec 6))
-(assert (and (= (bvsub v v) (_ bv53 6)) (not (bvssubo v v))))
+(assert (and (bvugt v (_ bv22 6)) (bvugt (_ bv0 6) u) (not (bvssubo v u))))
```

Comment:
What's the reason for changing this test?

Reply:
See https://github.com/cvc5/cvc5/pull/11115/files.
The thing is that the existing test does not exercise the elimination rewrite rule, because the benchmark is trivially solved regardless of the overflow operator.

I guess, if that PR is merged first, then this change is redundant here.